scripts/release/asset_publish.py

```diff
@@ -205,11 +205,12 @@ def validate_and_prepare_pipeline_component(
         )
 
         if registry and registry not in [PROD_SYSTEM_REGISTRY, registry_name]:
-            logger.log_error(
-                "Registry name for component's URI must be either "
-                + f"'{registry_name}' or '{PROD_SYSTEM_REGISTRY}'. Got '{registry}'"
+            logger.log_warning(
+                f"Dependent asset should exist in '{registry_name}' or '{PROD_SYSTEM_REGISTRY}'."
+                f" From component '{name}' URI we got: '{registry}'."
+                + f" Component publish would fail if release process does not have read access to '{registry}'."
+                + " Please contact to make sure, read access is provided inorder to be able to publish."
```

Comment:
Consider a simpler message:
```suggestion
                f"Dependencies should exist in '{registry_name}' or '{PROD_SYSTEM_REGISTRY}'. "
                f"The URI for component '{name}' references registry '{registry}', "
                "and publishing will fail if the release process does not have read access to it."
```

Reply:
Done. Thank you